src/traces/parcoords/parcoords.js

```diff
@@ -227,8 +234,6 @@ function styleExtentTexts(selection) {
     selection
         .classed('axisExtentText', true)
         .attr('text-anchor', 'middle')
-        .style('font-weight', 100)
-        .style('font-size', '10px')
```

Comment:
Ha. I see `10px` was used before. I should've caught that. Elsewhere in plotly.js this would be [`12px`](https://github.com/plotly/plotly.js/blob/460ac39ae99f0e3de46cb9307dc77f581ecbafcb/src/plots/layout_attributes.js#L23).

I guess we'll have to keep it this way until v2 :grimacing: